.gitignore

```diff
@@ -3,7 +3,7 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
```

Comment:
The `.venv/` entry is correctly included to ignore the virtual environment directory. Ensure that this matches the actual directory name used in your project for the virtual environment. If your virtual environment directory has a different name, update this entry accordingly.